src/Apps/Settings/Routes/SavedSearchAlerts/Components/SavedSearchAlertEditFormPlaceholder.tsx

```diff
@@ -1,20 +1,49 @@
 import {
-  Box,
   Flex,
   Text,
   Spacer,
   SkeletonText,
   SkeletonBox,
+  Skeleton,
 } from "@artsy/palette"
+import { AlertProvider } from "Components/Alert/AlertProvider"
 import { CriteriaPillsPlaceholder } from "Components/Alert/Components/CriteriaPills"
+import { Modal } from "Components/Alert/Components/Modal/Modal"
 import { Media } from "Utils/Responsive"
 
-export const SavedSearchAlertEditFormPlaceholder = () => {
+export const SavedSearchAlertEditFormPlaceholder: React.FC<{
+  onCloseClick?: () => void
+}> = ({ onCloseClick }) => {
   return (
-    <Box flex={1} p={4}>
-      <Text variant="lg" mb={4}>
-        Edit Alert
-      </Text>
+    <>
+      <Media greaterThanOrEqual="md">
+        <Skeleton flex={1} p={4}>
+          <SavedSearchAlertEditFormPlaceholderContext />
+        </Skeleton>
+      </Media>
+      <Media lessThan="md">
+        <AlertProvider isEditMode>
+          <Skeleton flex={1} p={4}>
+            <Modal onClose={onCloseClick} backgroundColor={"white100"}>
+              <Skeleton px={2}>
+                <SavedSearchAlertEditFormPlaceholderContext />
+              </Skeleton>
+            </Modal>
+          </Skeleton>
+        </AlertProvider>
+      </Media>
```

Comment:
Placeholder for the Edit Alert view was broken, fixing it in this PR by adding `Modal` together with `AlertProvider` to the Placeholder